packages/meteor-gazelle-createdby/lib/behavior/events.js

```diff
@@ -0,0 +1,44 @@
+// Adapted from https://github.com/jagi/meteor-astronomy-timestamp-behavior/
+events = {};
+
+events.beforeInsert = function () {
+  // Find a class on which the behavior had been set.
+  var behaviorData = Astro.utils.behaviors.findBehavior(this.constructor,
+                                                        'createdby');
+
+  // If the "hasCreatedByField" option is set.
+  if (behaviorData.hasCreatedByField) {
+    // Set value for createdBy field.
+    this.set(behaviorData.createdByFieldName, Meteor.userId());
+  }
```

Comment:
We should also add updatedBy here for consistency, so we always have a non-null updatedBy field.